src/Web3.elm

```diff
@@ -182,6 +184,87 @@ toChecksumAddress (Address address) =
 -- CORE
 
 
+toWei : EthDenomination -> BigInt -> BigInt
+toWei unit amount =
+    mul amount (getValueOfUnit unit)
+
+
+getValueOfUnit : EthDenomination -> BigInt
+getValueOfUnit unit =
+    case unit of
+        Wei ->
+            (fromInt 1)
+
+        Kwei ->
+            (fromInt 1000)
+
+        Ada ->
+            (fromInt 1000)
+
+        Femtoether ->
+            (fromInt 1000)
+
+        Mwei ->
+            (fromInt 1000000)
+
+        Babbage ->
+            (fromInt 1000000)
+
+        Picoether ->
+            (fromInt 1000000)
+
+        Gwei ->
+            (fromInt 1000000000)
+
+        Shannon ->
+            (fromInt 1000000000)
+
+        Nanoether ->
+            (fromInt 1000000000)
+
+        Nano ->
+            (fromInt 1000000000)
+
+        Szabo ->
+            (fromInt 1000000000000)
+
+        Microether ->
+            (fromInt 1000000000000)
+
+        Micro ->
+            (fromInt 1000000000000)
+
+        Finney ->
+            (fromInt 1000000000000000)
+
+        Milliether ->
+            (fromInt 1000000000000000)
+
+        Milli ->
+            (fromInt 1000000000000000)
+
+        Ether ->
+            (fromInt 1000000000000000000)
+
+        Kether ->
+            mul (fromInt 1000000000000000000) (fromInt 1000)
```

Comment:
This is just because of integer overflow in javascript. Could have used a `fromString` but that would have introduced an `Maybe` and made things more complicated.